lesson-5.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+    <link rel="stylesheet" href="css/lesson-5.css">
+    <div class="container_about">
+        <p class="heading_about">About</p>
+        <p class="text">Now Triphouse works</p>
+        <p class="text">Careers</p>
+        <p class="text">Privacy</p>
+        <p class="text">Terms</p>
+    </div>
+    <div class="container_property_types">
+        <p class="heading_property">Property types</p>
+        <p class="text">Guest houses</p>
+        <p class="text">Hotels</p>
+        <p class="text">Apartments</p>
+        <p class="text">Villas</p>
+        <p class="text">Holiday homes</p>
+        <p class="text">Hostels</p>
+    </div>
+    <div class="container_support">
+        <p class="heading_support">Support</p>
+        <p class="text">Contact Customer Service</p>
+        <p class="text">FAQ</p>
+    </div>
+    </div>
+    <footer>&copy; 2020 Triphouse, lnc. All right reserved</footer>
```

Comment:
I would use the footer tag for this whole section and the copyright section with div or p